### ISSUE TITLE
[Windows Terminal - Settings]: In dark mode, Luminosity ratio for 'Reset to inherited value' icon is '2.6:1', which is less than required '3:1'.

### ISSUE THREAD

**Issue by SaurabhNew** [microsoft/terminal#17902]
### Windows Terminal version

1.22.2362.0

### Windows build number

27695.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 27695.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Prerequisites:**
Settings> Personalization> Colors> Choose your Mode> Dark Mode.

**Repro steps:**
1. Open the 'windows terminal' application.
2. Apply dark mode by following the prerequisites.
3. Now open 'Setting' and activate 'Startup' tab.
4. Navigate to the 'Defaults' tab and activate it.
5. Now navigate to the 'Appearance' button and activate it.
6. Now navigate to any control like 'Line height' and change value.
7. 'Reset to inherited value' button will appear.
8. Open Accessibility Insights for windows and check contrast ratio of 'Reset to inherited value' icon. 
9. Observe the issue.

**User experience:**
Low vision users will find difficulty in tracking the control, if contrast ratio for the control's icon is less than '3:1' (Required).

**WCAG Reference Link:**
[[MAS 4.3.1 – No Disruption of Accessibility Features.docx (sharepoint.com)](https://microsoft.sharepoint.com/:w:/s/accessibility/EegntkLK4KBBvzE1qsKpIoABG2UIxUY2y3uo1LomKoz_bg?e=wLbDEr)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)

**Attachment:**
[MAS1.4.11 - In Dark mode, cca failed for 'Reset to inherited value' button..zip](https://github.com/user-attachments/files/16959865/MAS1.4.11.-.In.Dark.mode.cca.failed.for.Reset.to.inherited.value.button.zip)

### Expected Behavior

In dark mode, Luminosity ratio for 'Reset to inherited value' icon should be greater than or equal to 3:1. 

### Actual Behavior

In dark mode, Luminosity ratio for 'Reset to inherited value' icon is '2.6:1', which is less than required '3:1'.

**Note:**
Same issue repro for all 'Reset to inherited value' button's icon throughout.

**Comment by carlos-zamora** [comment]
@SaurabhNew is there an associated severity rating for this issue?

**Comment by carlos-zamora** [comment]
@SaurabhNew I'm making the change in PR #17912. I have a question though. The accent color is configurable by the user to be just about any color (I noticed you can't set it to white, thankfully). Because of that, the contrast can be wildly different. I've tested this out with various shades of the accent color (as described in [these docs](https://learn.microsoft.com/en-us/windows/apps/design/style/color#accent-color-palette)).

We want to maintain that the color used is the accent color, but going too light/dark removes that feeling of personalization from the app. Is there a suggested range of accent colors that are expected to be supported? Or any limitations on this space (like using the blue color you have it set to in your testing)?

**Comment by SaurabhNew** [comment]
@carlos-zamora, I tried to find the similar controls in native applications but haven't find anywhere.
